.evergreen/config.yml

```diff
@@ -23127,7 +23127,7 @@ buildvariants:
   - name: .ocsp-openssl-1.0.1
     distros:
     - ubuntu1804-test
-  batchtime: 20160
```

Comment:
See the top of this file. We don't modify this file directly but instead rely on the python scripts in `mongo-c-driver/build/evergreen_config_lib` to change it for us. In `variants.py` (see line 638), the batch times are in days as well so you shouldn't have to do any calculations from days to seconds directly.

Reply:
Ah sounds good, I've modified `variants.py` instead of modifying `config.yml` directly.

After offline discussion, I also lowered the TSAN and Valgrind tests' `batchtime`s to 1 day instead of 1 week.